src/v2/Apps/Home/Components/HomeCurrentFairsRail.tsx

```diff
@@ -49,8 +48,9 @@ const HomeCurrentFairsRail: React.FC<HomeCurrentFairsRailProps> = ({
           return (
             <RouterLink
               to={fair.href}
-              textDecoration="none"
               key={index}
+              textDecoration="none"
+              style={{ display: "block " }}
```

Comment:
(no display mixin)

Reply:
Weird it should be there? https://github.com/artsy/force/blob/53843fa49965a1fc7570ad9bb4806153f1740800/src/v2/System/Router/RouterLink.tsx#L3

Reply:
Looks like VSCode was buggin, will fix 👍